src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,63 @@
 
 import org.springframework.stereotype.Service;
 
```

Comment:
3. **Error handling**: Consider adding more descriptive error messages to differentiate between different scenarios of input validation or calculation errors.